Add spell icon zoom option

ABangingDonk opened the pull request:
Provide an option to 'zoom' the spell icons (globally) of spell frames, removing the grey borders

ABangingDonk committed:
Add spell icon zoom option

Provide an option to 'zoom' the spell icons (globally) of spell frames, removing the grey borders

DragonKarsh reviewed (changes requested):
I like this. Would you be able to make it so that the zoom is a slider and you can set the percentage to zoom? Rather than it be a toggle and it zoom to a specific value?

ABangingDonk committed:
Make spell icon zoom configurable

Instead of hardcoding a zoomed in state for spell icons, make it configurable via slider in the options menu

DragonKarsh commented on EventHorizon/Frame/MainFrame.lua:
```diff
@@ -302,6 +302,11 @@ function MainFrame:UpdateAllFrames()
 			:GetIcon()
 			:SetSize(height, height)
+			
+			local coordScale = (EventHorizon.opt.zoom) / 100
```
Need to set a default value for zoom in the default profile so that this does not cause an arithmetic error when the value is nil. Once you get that in, I will merge

ABangingDonk replied:
Good spot, thanks :)

DragonKarsh reviewed (changes requested):
Need zoom to have a default value set on the profile in Core/Database.lua

ABangingDonk committed:
Give spell icon zoom a default

Set default spell icon zoom to 0 for new profiles and handle cases where existing profiles are missing zoom preference

DragonKarsh reviewed (approved):
Very nice
